research-software-directory/RSD-as-a-service
# feat: deep linking edit project pages

# Enable deep linking of edit project pages

Closes #815

Changes proposed in this pull request:
* Enables deep linking of edit project pages. Each page has it's own url.    
* Root route is forwarded to information page (first page)
* Deep link to team members is added to admin rsd-contributors link
* User consent OK button label changed from OK to Accept.

How to test:
* `make start` to build app
* Login and create new project
* After creation you should be redirected to projects/[slug]/information
* Navigate between all project sections. Confirm that each section has unique url path (you can save as shortcut)
* Login as rsd admin. 
* Navigate to rsd contributors section. 
* Confirm that project contributors link sends you directly to edit team member page

PR Checklist:
